tests/testthat/test-label-ordinal.txt

```diff
@@ -0,0 +1,10 @@
+> ordinal(c(1:4, 11:21, 101, NA))
+ [1] "1st"   "2nd"   "3rd"   "4th"   "11th"  "12th"  "13th"  "14th"  "15th" 
+[10] "16th"  "17th"  "18th"  "19th"  "20th"  "21st"  "101st" NA     
+
+> ordinal(c(1, 2, 10, 11, NA), rules = ordinal_french())
+[1] "1er" "2e"  "10e" "11e" NA   
+
+> ordinal(c(1, 2, 10, 11, NA), rules = ordinal_french("f", TRUE))
+[1] "1res" "2es"  "10es" "11es" NA    
```

Comment:
Is there a reference for this ? That looked odd at first sight, but then I don't know how I would abbreviate.

Reply:
I found these references
http://www.guide-typographie.com/ab_nombreordinaux.htm
http://www.academie-francaise.fr/abreviations-des-adjectifs-numeraux